src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +70,8 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating() {
```

Comment:
Declaring a default implementation for the `isMutating()` helps to reduce the amount of code. However, choosing an arbitrary default value for isMutating (either true or false) is not recommended as it reduces code understandability. You could consider refactoring the class and this method `abstract`, and force only the child classes to implement instead.